tasks/main.yml

```diff
@@ -21,9 +21,9 @@
   when: not elasticsearch_installed.stat.exists
 
 - name: Install ES plugins
-  shell: /usr/share/elasticsearch/bin/plugin {% if elasticsearch_file_name|get_major_version == 1 %}--{% endif %}remove {{ item }} && /usr/share/elasticsearch/bin/plugin {% if elasticsearch_file_name|get_major_version == 1 %}--{% endif %}install {{ item }}
-  when: head_plugin_path.stat.exists != true
+  shell: /usr/share/elasticsearch/bin/plugin {% if elasticsearch_file_name|get_major_version == 1 %}--{% endif %}install {{ item }}
   with_items: "{{ plugins_to_install }}"
+  ignore_errors: True
```

Comment:
This will ignore issues with trying to install an existing plugin, but I'm wary of it also ignoring errors we might want to know about, like legit "could not install the plugin" errors. Is there a way we can be more explicit about which errors we're ok with?